src/itest/java/com/iexec/blockchain/ThisAppClient.java

```diff
@@ -0,0 +1,24 @@
+package com.iexec.blockchain;
+
+import com.iexec.common.chain.adapter.args.TaskContributeArgs;
+import com.iexec.common.chain.adapter.args.TaskFinalizeArgs;
+import com.iexec.common.chain.adapter.args.TaskRevealArgs;
+import feign.Param;
+import feign.RequestLine;
+
+interface ThisAppClient {
```

Comment:
`ApiUnderTest ` or `BlockchainAdapterApi` ??

Reply:
I agree, updated to `BlockchainAdapterApiClient`